### PR TITLE
I can be merged now.

### PR DESCRIPTION
**Pull request by mstackhouse**
## Types of changes

- [ ] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)
- [x] I have read the **CONTRIBUTING** document.
- [x] My code follows the code style of this project.
- [x] My change requires a change to the documentation.
- [x] I have updated the documentation accordingly.
- [x] I have added tests to cover my changes.
- [x] All new and existing tests passed.

### CODE REVIEW

**Comment by mstackhouse** [comment]
Only these files need to be reviewed: 

- [ ] https://github.com/atorus-research/Tplyr/pull/163/files#diff-b1d304bc2e54816287509e162617e7843839f88f79045d41b85802a7ac474b9d
- [ ] https://github.com/atorus-research/Tplyr/pull/163/files#diff-850aa2dd10723b0c29f075c2acf08b0631058db0f18028dfe34d2cf2cb60c3ec
- [ ] https://github.com/atorus-research/Tplyr/pull/163/files#diff-cd0be89dd65e80434332204a0f3469519558d136a80b08507869de35c67ca9f7

File: R/collapse_row_labels.R [hunk 1]
```diff
@@ -0,0 +1,122 @@
...
+#'   "A",         "C",         "H",         "O",    3L,
+#'   "A",         "D",         "H",         "P",    4L,
+#'   "A",         "D",         "I",         "Q",    5L,
+#'   "A",         "D",         "I",         "R",    6L,
+#'   "B",         "E",         "J",         "S",    7L,
+#'   "B",         "E",         "J",         "T",    8L,
+#'   "B",         "E",         "K",         "U",    9L,
+#'   "B",         "F",         "K",         "V",   10L,
+#'   "B",         "F",         "L",         "W",   11L
+#' )
```
Comment (ShiyuC) [inline R/collapse_row_labels.R:43]:
```suggestion
#' ~row_label1, ~row_label2, ~row_label3, ~row_label4, ~var1,
#'   "A",         "C",         "G",         "M",        1L,
#'   "A",         "C",         "G",         "N",        2L,
#'   "A",         "C",         "H",         "O",        3L,
#'   "A",         "D",         "H",         "P",        4L,
#'   "A",         "D",         "I",         "Q",        5L,
#'   "A",         "D",         "I",         "R",        6L,
#'   "B",         "E",         "J",         "S",        7L,
#'   "B",         "E",         "J",         "T",        8L,
#'   "B",         "E",         "K",         "U",        9L,
#'   "B",         "F",         "K",         "V",        10L,
#'   "B",         "F",         "L",         "W",        11L
#' )
```

File: tests/testthat/test-collapse_row_labels.R [hunk 2]
```diff
@@ -0,0 +1,70 @@
...
+  "A",         "C",         "H",         "O",    3L,
+  "A",         "D",         "H",         "P",    4L,
+  "A",         "D",         "I",         "Q",    5L,
+  "A",         "D",         "I",         "R",    6L,
+  "B",         "E",         "J",         "S",    7L,
+  "B",         "E",         "J",         "T",    8L,
+  "B",         "E",         "K",         "U",    9L,
+  "B",         "F",         "K",         "V",   10L,
+  "B",         "F",         "L",         "W",   11L
+)
```
Comment (ShiyuC) [inline tests/testthat/test-collapse_row_labels.R:14]:
```suggestion
dat <- tibble::tribble(
  ~row_label1, ~row_label2, ~row_label3, ~row_label4, ~var1,
  "A",         "C",         "G",         "M",         1L,
  "A",         "C",         "G",         "N",         2L,
  "A",         "C",         "H",         "O",         3L,
  "A",         "D",         "H",         "P",         4L,
  "A",         "D",         "I",         "Q",         5L,
  "A",         "D",         "I",         "R",         6L,
  "B",         "E",         "J",         "S",         7L,
  "B",         "E",         "J",         "T",         8L,
  "B",         "E",         "K",         "U",         9L,
  "B",         "F",         "K",         "V",         10L,
  "B",         "F",         "L",         "W",         11L
)
```

**Review by ShiyuC — COMMENTED** [review]
Looks good to me! I just had super minor suggestions on the alignment of the tibbles

**Review by asbates — APPROVED** [review]

**Pull request merged**